CODEOWNERS

```diff
@@ -34,8 +34,6 @@ provider/datagen/                 @sffc @robertbastian @Manishearth
 provider/fs/                      @sffc
 provider/macros/                  @Manishearth @sffc
 tools/benchmark/binsize/          @gnrunge
-tools/benchmark/macros/           @gregtatum
-tools/benchmark/memory/           @gregtatum
```

Comment:
Hi, Greg! Can you suggest new owners for these components? We ideally shouldn't just remove these components from the owners file with no replacement.

Reply:
I've not been involved with the project for awhile as I've been focusing on Firefox Translations, so I'm not sure who is working in the space who would be good replacements.

Reply:
Ok, I don't really have a proposal either but in this case these components are small and haven't changed a whole lot since you wrote them so I think it's fine for them to fall back to the global owners. For larger crates, ideally part of the offboarding process should be for contributors to proactively transfer ownership.